packages/waku/src/lib/renderers/rsc-renderer.ts

```diff
@@ -9,10 +9,11 @@ import type {
 } from '../../server.js';
 import type { ResolvedConfig } from '../config.js';
 import { filePathToFileURL } from '../utils/path.js';
-import { parseFormData } from '../utils/form.js';
-import { streamToArrayBuffer, arrayBufferToString } from '../utils/stream.js';
+import { streamToArrayBuffer } from '../utils/stream.js';
 import { decodeActionId } from '../renderers/utils.js';
 
+import { bufferToFormData, bufferToString } from 'hono/utils/buffer';
+
```

Comment:
use Hono directly

Reply:
This file should not depend on Hono. It can be used without Hono.

Reply:
This file should not depend on Hono. It can be used without Hono.